gtsfm/frontend/correspondence_generator/keypoint_aggregator/keypoint_aggregator_unique.py

```diff
@@ -0,0 +1,64 @@
+"""Keypoint aggregator that assumes each detected keypoint per image pair will be unique across he entire image.
+
+Authors: John Lambert
+"""
+
+from typing import Dict, List, Optional, Tuple
+
+import numpy as np
+
+from gtsfm.common.keypoints import Keypoints
+from gtsfm.frontend.correspondence_generator.keypoint_aggregator.keypoint_aggregator_base import KeypointAggregatorBase
+
+
+class KeypointAggregatorUnique(KeypointAggregatorBase):
```

Comment:
Maybe `KeypointArregatorNoDedup`

Reply:
Acknowledged. I think I prefer `KeypointAggregatorUnique` a bit more, since we assume keypoint detections in each image pair are unique. I don't love either name, though.